cluster/service/cluster_service_blackbox_test.go

```diff
@@ -630,6 +631,54 @@ func (s *ClusterServiceTestSuite) TestListForAuth() {
 	})
 }
 
+func (s *ClusterServiceTestSuite) TestDelete() {
+
+	s.T().Run("ok", func(t *testing.T) {
+		// given
+		c := test.CreateCluster(t, s.DB)
+		sa := &authtestsupport.Identity{
+			Username: auth.ToolChainOperator,
+			ID:       uuid.NewV4(),
+		}
+		ctx, err := authtestsupport.EmbedServiceAccountTokenInContext(context.Background(), sa)
+		require.NoError(t, err)
+		// when
+		err = s.Application.ClusterService().Delete(ctx, c.ClusterID)
```

Comment:
The delete test should be a bit more complicated.
1. Add another cluster + cluster identity as a noise
2. Add some cluster identities for the cluster which is about to be deleted
3. Delete the cluster
4. Check the cluster and all its identities are gone
5. Check that the noise data (the other cluster and its identities) are still there

Reply:
done in 870041f